scripts/cross-microblazeel-zephyr-elf.txt

```diff
@@ -0,0 +1,20 @@
+[binaries]
+# Meson 0.53.2 doesn't use any cflags when doing basic compiler tests,
+# so we have to add -nostdlib to the compiler configuration itself or
+# early compiler tests will fail. This can be removed when picolibc
+# requires at least version 0.54.2 of meson.
+c = ['microblazeel-zephyr-elf-gcc', '-nostdlib', '-mxl-soft-mul', '-mxl-soft-div', '-msoft-float']
```

Comment:
are these compiler flags for math operations not provided in a multilib configuration?

Reply:
Seems it does. I removed those options. I have very little trust for microblaze-gcc...

```
microblazeel-zephyr-elf-gcc --print-multi-lib
.;
bs;@mxl-barrel-shift
m;@mno-xl-soft-mul
le;@mlittle-endian
m/mh;@mno-xl-soft-mul@mxl-multiply-high
m/le;@mno-xl-soft-mul@mlittle-endian
m/mh/le;@mno-xl-soft-mul@mxl-multiply-high@mlittle-endian
bs/m;@mxl-barrel-shift@mno-xl-soft-mul
bs/le;@mxl-barrel-shift@mlittle-endian
bs/m/mh;@mxl-barrel-shift@mno-xl-soft-mul@mxl-multiply-high
bs/m/le;@mxl-barrel-shift@mno-xl-soft-mul@mlittle-endian
bs/m/mh/le;@mxl-barrel-shift@mno-xl-soft-mul@mxl-multiply-high@mlittle-endian
```